src/libs/SidebarUtils.js

```diff
@@ -283,6 +285,7 @@ function getOptionData(report, reportActions, personalDetails, preferredLocale,
     result.isArchivedRoom = ReportUtils.isArchivedRoom(report);
     result.isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
     result.isExpenseRequest = ReportUtils.isExpenseRequest(report);
+    result.isIOURequest = ReportUtils.isIOURequest(report);
```

Comment:
I don't think it a good idea to add a new item to the `result` when it's not used in the component. So, I think in this case, we better use const instead of assigning the value to the option item.